packages/frontend/src/translations/en.global.json

```diff
@@ -1700,6 +1700,21 @@
             "title": "Enter Two-Factor Verification Code"
         }
     },
+    "twoFactorDisbleBanner": {
+        "desc": "NEAR wallet will soon remove support for two-factor authentication, please disable 2FA for your connected accounts.",
```

Comment:
<img width="228" alt="image" src="https://user-images.githubusercontent.com/25015977/192899440-5132917f-cf38-4b3a-be31-84ed514a68a2.png">

In latest copy of Figma, Jacob made a minor change in the text here